### PR TITLE
Slash commands

### PR DESCRIPTION
**Pull request by Psykka**

### CODE REVIEW

**Comment by Psykka** [comment]
needs to add react response to other commands

File: src/commands/music/loop.js [hunk 1]
```diff
@@ -14,24 +34,28 @@ export default class Loop extends Command {
...
 
-    switch (args[0].toUpperCase()) {
+    if (isInteraction) {
+      type = options.getString('type')
+    } else {
+      if (!args[0]) return channel.send('Você precisa me dizer o tipo de loop')
+      type = args[0].toUpperCase()
+    }
+
+    switch (type.toUpperCase()) {
```
Comment (Psykka) [inline src/commands/music/loop.js]: Don't need toUpperCase

Comment (Psykka) [reply src/commands/music/loop.js]: Fixed

**Comment by derpyxd** [comment]
LGTM 👍

**Pull request merged**